src/commands/user/lookup.ts

```diff
@@ -1,13 +1,66 @@
-import { Message } from 'discord.js'
-
-// Why "_aruments" and not "arguments"?
-// Because typescript was going nuts over the fact that "arguments" is a thing, I dont know why.
-export function lookUp(
-	message: Message,
-	_arguments: any[]
-): void | Promise<Message<boolean>> {
-	if (!_arguments)
-		return message.channel.send('We require the username argument!');
-	
-	
-}
+import fetch from 'node-fetch';
+import { Message, MessageEmbed, MessageSelectMenu } from 'discord.js'
+
+var URLPattern = new RegExp('^(https?:\\/\\/)?'+
```

Comment:
DONT USE VAR

We use typescript for a reason. Thus use `let` and `const`.

Also what the fuck is this regex

Reply:
Also pretty sure there's url builders builtin js.

Reply:
^^